suite-native/atoms/src/Button/Button.tsx

```diff
@@ -227,9 +238,32 @@ export const ButtonIcon = ({ iconName, color, buttonSize }: ButtonIconProps) =>
     <Icon name={iconName} color={color} size={buttonToIconSizeMap[buttonSize]} />
 );
 
+export const isIconName = (element: ButtonAccessoryType) =>
+    G.isNotNullable(element) &&
+    typeof element === 'string' &&
+    icons[element as IconName] !== undefined;
+
+// ButtonAccessoryView renders either a ButtonIcon or a provided custom element
+// iconColor and iconSize are only used when element is an IconName
+export const ButtonAccessoryView = ({
+    element,
+    iconColor = 'iconDefault',
```

Comment:
What about "hiding" these defaults inside of IconButton component? It should be its responsible IMO. Just idea.

Reply:
0c52c10957